src/backend/metatypes.h

```diff
@@ -163,9 +117,9 @@ namespace snmalloc
      * assert that this chunk is being used as a slab (i.e., has an associated
      * owning allocator).
      */
-    [[nodiscard]] SNMALLOC_FAST_PATH MetaCommon* get_meta() const
+    [[nodiscard]] SNMALLOC_FAST_PATH BackendMetadata* get_meta() const
     {
-      return reinterpret_cast<MetaCommon*>(meta & ~META_BOUNDARY_BIT);
+      return reinterpret_cast<BackendMetadata*>(meta & ~META_BOUNDARY_BIT);
```

Comment:
Should this use unsafe_from_uintptr for symmetry with the constructor?

Reply:
It should, though I'd rather that we had a generic IntPtrPair class like LLVM.  This code is inherited from the existing code though.

Reply:
Done.